patches/web_contents.patch

```diff
@@ -15,30 +15,14 @@ index cf3733f..065df39 100644
    WebContentsViewDelegate* delegate =
        GetContentClient()->browser()->GetWebContentsViewDelegate(this);
  
-@@ -1586,6 +1592,7 @@ void WebContentsImpl::Init(const WebContents::CreateParams& params) {
+@@ -1591,6 +1597,7 @@ void WebContentsImpl::Init(const WebContents::CreateParams& params) {
      view_.reset(CreateWebContentsView(this, delegate,
                                        &render_view_host_delegate_view_));
    }
 +  }  // !view_
  
    if (browser_plugin_guest_ && !GuestMode::IsCrossProcessFrameGuest(this)) {
      view_.reset(new WebContentsViewGuest(this, browser_plugin_guest_.get(),
-@@ -4858,6 +4865,7 @@ void WebContentsImpl::RendererUnresponsive(
-   // We might have been waiting for both beforeunload and unload ACK.
-   // Check if tab is to be unloaded first.
-   if (rfhi->IsWaitingForUnloadACK()) {
-+#if 0
```

Comment:
This exact patch is no longer applicable, the code around have been refactored.
I would suggest to continue the discussion in the new pr (#313).